packages/p/pqp/xmake.lua

```diff
@@ -0,0 +1,28 @@
+package("pqp")
+
+    set_homepage("http://gamma.cs.unc.edu/SSV/")
+    set_description("A Proximity Query Package")
+
+    set_urls("http://gamma.cs.unc.edu/software/downloads/SSV/pqp-$(version).tar.gz",
+             "https://github.com/GammaUNC/PQP.git")
+
+    add_versions("1.3", "00bdcb380206393349ed7ee9a773f894bf796059c66cec0d97c0a93649eb012f")
+    add_patches("1.3", path.join(os.scriptdir(), "patches", "fix.patch"), "9160f8bc35d23dd5a84af0891410fafa07b8fd9329389724627a68e96b4c270f")
+
+    on_install(function (package)
+        io.writefile("xmake.lua", [[
+            add_rules("mode.debug", "mode.release")
+            target("pqp")
+                set_kind("$(kind)")
+                add_files("**.cpp|**/demos/**")
+                add_headerfiles("**.h|**/demos/**")
```

Comment:
```lua
add_files("src/*.cpp")
add_headerfiles("src/*.h")
```

Reply:
Unfortunatly this isn't possible, because the src folder is inside the PQP_v1.3 folder. But I could do it like this:
```lua
add_files("**/src/*.cpp")
add_headerfiles("**/src/*.h")
```